client/src/components/Login.js

```diff
@@ -1,4 +1,5 @@
-import React from "react";
+import React, { useState } from "react";
+import { signApi } from "../api/auth";
 import Avatar from "@material-ui/core/Avatar";
```

Comment:
Also, I noticed that when I run that locally it's failing complaining that it cannot find `@material-ui/core/Avatar`. Probably you have installed that either globally `npm install -g ....` or without saving it to the client's dependencies `npm install --save ....`.
Make sure that you have all the dependencies you are introducing installed like that one.